lib/new_inject.rb

```diff
@@ -0,0 +1,10 @@
+class Array
+  def new_inject(*args)
+    copy = self.dup
+    symbol = args.delete(args.select{|a|a.is_a?(Symbol)}.first)
```

Comment:
Space after closing `|` missing.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.